src/renderer/components/outputs/LargeImageOutput.tsx

```diff
@@ -0,0 +1,176 @@
+/* eslint-disable no-nested-ternary */
+import { ViewOffIcon, WarningIcon } from '@chakra-ui/icons';
+import { Box, Center, HStack, Image, Spinner, Text, useColorModeValue } from '@chakra-ui/react';
+import { memo, useEffect, useState } from 'react';
+import { useContext, useContextSelector } from 'use-context-selector';
+import { NamedExpression, NamedExpressionField } from '../../../common/types/expression';
+import { NumericLiteralType } from '../../../common/types/types';
+import { GlobalContext, GlobalVolatileContext } from '../../contexts/GlobalNodeState';
+import { OutputProps } from '../inputs/props';
+
+interface LargeImageBroadcastData {
+    image: string;
+    width: number;
+    height: number;
+    channels: number;
+}
+
+export const LargeImageOutput = memo(
+    ({ id, outputId, useOutputData, animated = false, schemaId }: OutputProps) => {
+        const inputDataChanges = useContextSelector(
+            GlobalVolatileContext,
+            (c) => c.inputDataChanges
+        );
+        const lastInputDataUpdatedId = useContextSelector(
+            GlobalVolatileContext,
+            (c) => c.lastInputDataUpdatedId
+        );
+
+        const [stale, setStale] = useState(false);
+
+        const zoom = useContextSelector(GlobalVolatileContext, (c) => c.zoom);
+
+        const value = useOutputData(outputId) as LargeImageBroadcastData | undefined;
+
+        const { setManualOutputType, schemata } = useContext(GlobalContext);
+
+        const schema = schemata.get(schemaId);
+
+        useEffect(() => {
+            if (value) {
+                setStale(false);
+            }
+        }, [value]);
+
+        useEffect(() => {
+            if (value && lastInputDataUpdatedId !== id) {
+                setStale(true);
+            }
```

Comment:
This just doesn't work. If you create a simple chain with 2 Load Image nodes, and set an image for both of them, one will always be marked stale.

Reply:
Oh yeah...... Got any ideas for how to do that better?

Reply:
Ok, what i've done is just not show the stale indicator when it's a starting node. I think that's good enough